scripts/run_backend_tests.py

```diff
@@ -169,68 +171,71 @@ def __init__(self, test_target, generate_coverage_report):
 
     def run(self):
         """Runs all tests corresponding to the given test target."""
+        env = os.environ.copy()
         test_target_flag = '--test_target=%s' % self.test_target
         if self.generate_coverage_report:
             exc_list = [
-                sys.executable, COVERAGE_MODULE_PATH, 'run', '-p',
+                sys.executable, COVERAGE_MODULE_PATH, 'run',
                 TEST_RUNNER_PATH, test_target_flag
             ]
+            rand = random.Random(os.urandom(8)).randint(0, 999999)
+            data_file = '.coverage.%s.%s.%06d' % (
+                socket.gethostname(), os.getpid(), rand)
+            env['COVERAGE_FILE'] = data_file
+            python_utils.PRINT('Coverage data for %s is in %s' % (
```

Comment:
Should we use concurrent_task_util.log here? To avoid having such log:
![image](https://user-images.githubusercontent.com/16653571/140534312-243a35b0-d5ff-44ae-82de-50fcd414b3c6.png)

Reply:
Also so you know the infinite number of `[datastore]` in the log?

Reply:
done. I don't know where the `[datastore]` is coming from. It appears in other CI runs though, so it's not introduced by this PR

Reply:
> I don't know where the [datastore] is coming from. It appears in other CI runs though, so it's not introduced by this PR

Thanks for the info, I agree that it is already there, it would be great if we can look into it and avoid such a log as it's too long and makes it harder to navigate (This comment isn't blocking this PR.)